application/select-display.sh

```diff
@@ -0,0 +1,50 @@
+set -euo pipefail
+
+SCALING_PREF=$(cat /var/lib/gui-localization/screen-scaling 2>/dev/null || echo "default")
+readonly SCALING_PREF
+
+CONNECTED_OUTPUTS=$(xrandr | grep ' connected' | awk '{ print $1 }')
+readonly CONNECTED_OUTPUTS
+
+echo -e "Connected outputs:\n$CONNECTED_OUTPUTS\n"
+
+scaling_pref_params=""
+
+echo "Using scaling preference '$SCALING_PREF'"
+
+case "$SCALING_PREF" in
+    "default" | "full-hd")
+        scaling_pref_params=(--mode 1920x1080)
+        ;;
+    "native")
+        scaling_pref_params=(--auto)
+        ;;
+    *)
+        scaling_pref_params=(--auto)
+        ;;
+esac
+
+if [ -z "$CONNECTED_OUTPUTS" ]; then
+
+    echo "No connected outputs found. Attempting to apply xrandr globally."
+    xrandr --auto # this is kind of useless?
```

Comment:
From xrandr man pages:

```For disconnected but enabled outputs, this will disable them.```

So if all are disconnected, this will simply disable them. I suggest to replace this with a warning, unless there's some reason to do this.

Reply:
Removed this in b1d2db7